http-queries/routing.http

```diff
@@ -0,0 +1,40 @@
+# Variables defined in settings.json rest-client.environmentVariables:
+# (See https://marketplace.visualstudio.com/items?itemName=humao.rest-client#environment-variables or call me)
+# * @dopplerToken
+# * @apisHost
+
+@customDomain=midominio.com
+
+# Find avaliable options in `RuleType.cs`
+# (Both HTTP and HTTPS accept requests)
+@ruleType=HttpsAndHttp
+# (HTTP redirects to HTTPS)
+# @ruleType=HttpsOnly
+# (Only HTTP accepts requests)
+# @ruleType=HttpOnly
+
+# Find avaliable options in `appsettings > TraefikConfiguration > ServicesMapping`
+@service=relay-tracking
+# @service=relay-tracking-iis
+# @service=forms-landing
+
+###
+GET https://{{apisHost}}/routing/version.txt
```

Comment:
Can `routing` be a `.vscode/settings.json` variable ? We have `CUSTOM_DOMAIN_PREFIX_SEGMENT` in [Swarm](https://github.com/MakingSense/doppler-swarm/blob/master/custom-domain-stack/_post.env#L5)

Reply:
I am not sure, this value will never change inside this file. Each variable that we add, makes this file more cryptic and less intuitive. So, in my opinion, hardcoding `routing` is a good thing.

Reply:
Marking this as resolved